pkg/cloud/cloud_test.go

```diff
@@ -67,6 +67,22 @@ func TestCreateDisk(t *testing.T) {
 			},
 			expErr: nil,
 		},
+		{
+			name:       "success: normal with gp2 options",
```

Comment:
is it intentional to include in same PR? It loolks fine, but, want to make sure ur not accidentally committing something wip.

Reply:
Yeah.. Added this to improve some coverage on cloud.go we can remove this if you want to put it separately..

Reply:
fine to keep it here :+1: